packages/utils/src/time.ts

```diff
@@ -148,7 +90,7 @@ export const browserPerformanceTimeOrigin = ((): number | undefined => {
   // performance.timing.navigationStart, which results in poor results in performance data. We only treat time origin
   // data as reliable if they are within a reasonable threshold of the current time.
 
-  const { performance } = WINDOW;
+  const { performance } = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
```

Comment:
Can we make this a new type instead of this type cast?

Reply:
I think it's fine to keep esp because it's only being used here.